.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 sudo: false
 go:
-    - 1.9
+    - 1.12
```

Comment:
looks like this is now in line with agent
https://github.com/aws/amazon-ecs-agent/blob/master/.travis.yml#L5